assets/js/app/Panel.js

```diff
@@ -308,7 +431,12 @@ LocusZoom.Panel.prototype.setTitle = function(title){
     return this;
 };
 
-// Initialize a panel
+
+/**
+ * Initialize the panel
+ * TODO: Clarify uninitialized panel (eg data vs rendering?)
```

Comment:
Initialize is essentially the panel's first draw function, in that a panel is just a stage for data layers and their scaffolding (e.g. axis if applicable, the curtain/loader elements, the actual `<g>` tag inside which all the child data layers do their rendering, etc.). Panels also have a `render` function, but that's the "loop" that gets called when anything triggers a re-draw (it updates the already-initialized scaffolding display elements and triggers a redraw of child data layers if needed, among other things).

An uninitialized panel may have its *layout* initialized (by `initializeLayout`... overloaded term to be sure) which means the layout "agrees with itself" (e.g. has sane dimensions / origin / margins with respect to discrete and proportional values, as well has having datalayers defined in the layout instantiated as children.